core/templates/components/graph-services/graph-layout.service.spec.ts

```diff
@@ -950,56 +1022,62 @@ describe('Graph Layout Service', () => {
         id: 'State5',
         label: 'State5',
         height: 33.6,
-        width: 113.40000000000002,
+        width: 126,
         reachableFromEnd: false
       }
     };
-    // The expectedGraphBoundaries are calculated from the x0 and y0 values from
-    // nodeDataWithPositionValueInPixel, where leftEdge is
-    // minimum(nodeDataWithPositionValueInPixel[nodeId].x0 - BORDER_PADDING)
-    // of all nodes, and rightEdge is
-    // maximum(nodeDataWithPositionValueInPixel[nodeId].x0 + BORDER_PADDING +
-    // nodeDataWithPositionValueInPixel[nodeId].width) of all nodes. Similarly,
-    // bottomEdge and topEdge are calculated using y0 and node height.
-    let expectedGraphBoundaries1 = sgls.getGraphBoundaries(
+
+    let actualGraphBoundaries1 = sgls.getGraphBoundaries(
       nodeDataWithPositionValueInPixel1);
-    let expectedGraphBoundaries2 = sgls.getGraphBoundaries(
+    let actualGraphBoundaries2 = sgls.getGraphBoundaries(
       nodeDataWithPositionValueInPixel2);
-    let expectedGraphBoundaries3 = sgls.getGraphBoundaries(
+    let actualGraphBoundaries3 = sgls.getGraphBoundaries(
       nodeDataWithPositionValueInPixel3);
 
     // The width of graph calculated as difference b/w left and right edge.
```

Comment:
calculated --> is calculated 
b/w --> between (try not to use abbrevs)

Reply:
Done